docs/pephub/deployment.md

```diff
@@ -1,22 +1,48 @@
 # Deployment
 
-## Introduction
+This is the publicly available instance of [PEPhub](https://github.com/pepkit/pephub) provided by the Sheffield lab. 
```

Comment:
Maybe something like...

We provide a [publically available instance of pephub](https://pephub.databio.org) free to use. However, there are many reasons why you might wish to deploy your own instance. Because of this, we provide detailed instructions and docker containers to spin up your own instance of pephub.